docker-compose.yml

```diff
@@ -228,8 +237,8 @@ services:
     minio1:
         image: minio/minio:RELEASE.2021-06-17T00-10-46Z
         volumes:
-            - data1-1:/data1
-            - data1-2:/data2
+            - ./data/data1-1:/data1
```

Comment:
I noticed an error in the logs with the original volumes here that minio was out of space, so I figured its data directory should just be mounted into your working dir the same as the scheduler data. I know there is probably a way to specify the size of the named volume, but I don't know exactly how much extra minio needs or will need in their example so it seems simplest to just leave it as a local volume mount